rules/order.js

```diff
@@ -0,0 +1,101 @@
+module.exports = {
+  'order/properties-order': [
+    {
+      order: 'strict',
+      properties: [
+        'content',
+      ],
+    },
+    {
+      order: 'strict',
+      properties: [
+        'position',
+        'z-index',
+        'top',
+        'bottom',
+        'left',
+        'right',
+      ],
+    },
+    {
+      order: 'strict',
+      properties: [
+        'transform',
+        'transform-origin',
+      ],
+    },
+    {
+      order: 'strict',
+      properties: [
+        'box-sizing',
+        'display',
+        'overflow',
+        'flex',
+        'flex-grow',
+        'flex-shrink',
+        'flex-basis',
+        'order',
+        'align-self',
+        'flex-direction',
+        'flex-wrap',
+        'justify-content',
+        'align-items',
+        'align-content',
+        'height',
+        'width',
+        'margin',
+        'margin-top',
```

Comment:
I'd be fine with switching it, if we have to change some stuff to upgrade the dependency that's fine IMO